Assets/_Application/Layers/LayerTypes/GeoJsonLayers/GeoJSONPointLayer.cs

```diff
@@ -100,7 +114,8 @@ public void AddAndVisualizeFeature<T>(Feature feature, CoordinateSystem original
                 var newPointCollection = GeometryVisualizationFactory.CreatePointVisualization(point, originalCoordinateSystem, PointRenderer3D);
                 newFeatureVisualisation.Data.Add(newPointCollection);
             }
-            
+
+            newFeatureVisualisation.padding = Vector3.one * GetSelectionRange();
```

Comment:
is dit waarom padding public is? ik zou het dan een functie maken SetPadding() zodat het duidelijker is wat er gebeurt